projects/ui-components/lf-repository-browser/lf-repository-browser.component.ts

```diff
@@ -80,7 +81,22 @@ export class LfRepositoryBrowserComponent implements OnDestroy {
 
   @Input()
   openSelectedNodesAsync: () => Promise<void> = async () => {
-    await this.openSelectedItemsAsync();
+    if (this.selectedItems && this.selectedItems.length > 0) {
+      await this.openSelectedItemsAsync();
+    } else {
+      console.debug('No selected nodes to open');
+    }
+  };
+
+  @Input()
+  openFocusedNodeAsync: () => Promise<void> = async () => {
+    if (this.entryList?.currentFocusIndex !== undefined) {
+      const nodeToOpen = this.entryList.listItems[this.entryList.currentFocusIndex].value as LfTreeNode;
+      this.entryDblClicked.emit([nodeToOpen]);
```

Comment:
why are we getting the focused the node through the currentFocusIndex, instead of getting it the entryFocus event detail?

Reply:
makes sense, updated and testing now